### DISCUSSION TITLE
IBM dialects (DB2 / Informix)

### DISCUSSION THREAD

**Discussion by jaguarxii** [MoonStorm/FastCrud#162] [Ideas]
Hello, do you have plans to include the sql dialects from IBM?

best regards

**Comment by MoonStorm** [comment]
I personally don't have enough experience with those.  But if anyone wants to give it a go, dialects live in [Dapper.FastCrud/SqlBuilders/Dialects](https://github.com/MoonStorm/Dapper.FastCRUD/tree/master/Dapper.FastCrud/SqlBuilders/Dialects). Of course they need to be covered by tests 😉
